packages/twenty-front/src/modules/object-record/record-board/record-board-column/components/RecordBoardColumn.tsx

```diff
@@ -25,35 +27,34 @@ type RecordBoardColumnProps = {
 export const RecordBoardColumn = ({
   recordBoardColumnId,
 }: RecordBoardColumnProps) => {
-  const { columnsFamilySelector, recordIdsByColumnIdFamilyState } =
-    useRecordBoardStates();
-  const columnDefinition = useRecoilValue(
-    columnsFamilySelector(recordBoardColumnId),
+  const recordGroupDefinition = useRecoilValue(
+    recordGroupDefinitionFamilyState(recordBoardColumnId),
   );
 
-  const recordIds = useRecoilValue(
-    recordIdsByColumnIdFamilyState(recordBoardColumnId),
+  const recordRowIdsByGroup = useRecoilComponentFamilyValueV2(
```

Comment:
should be recordCardIdsByGroup here?

Reply:
just for me to double check: what is rowIds vs recordIds? is it actually recordIds? 

to me rowIds are only useful to navigate between records using the keyboard, I'm a bit surprise we have this on Kanban

Reply:
okay, I think we should rename them all to recordIds, it would be way clearer